### ISSUE TITLE
State - "Sinop" is missing in the Turkey

### ISSUE THREAD

**Issue by kadirgombel** [dr5hn/countries-states-cities-database#74]
It's missing in the states.json
```
{
        "id": null,
        "name": "Sinop Province",
        "country_id": 225,
        "country_code": "TR",
        "state_code": "57"
}

**Comment by dr5hn** [comment]
@kadirgombel Awesome thanks for reporting. I will fix it. :tada:

**Comment by dr5hn** [comment]
@kadirgombel Fixed in #75 🎉 . Please Verify.

**Comment by kadirgombel** [comment]
@dr5hn LGTM 👍
